docs/tutorials/intro_tutorial.rst

```diff
@@ -970,6 +984,8 @@ iteration).
 .. code:: python
 
     from mesa.batchrunner import batch_run
+**Note for Windows OS users:** We recommend changing the the parameter `number_processes = None` to `number_processes = 1.`
```

Comment:
This reads like a general recommendation that Windows users should better use `number_processes = 1`, which is not the case. The problem is specific to running in Jupyter on Windows.

Reply:
Probably should also explain that the `number_processes` is for setting for multiple CPUs.